exampleCourse/questions/template/checkbox/feedback/question.html

```diff
@@ -0,0 +1,15 @@
+<pl-question-panel>
+  <p>Select all of the following numbers that are prime.</p>
+</pl-question-panel>
+
+<pl-checkbox
+  answers-name="numbers"
+  number-answers="7"
+  fixed-order="true"
+  partial-credit="true"
+  min-correct="2"
+>
+  {{#params.options}}
+    <pl-answer correct="{{correct}}" feedback="{{feedback}}">{{answer}}</pl-answer>
+  {{/params.options}}
```

Comment:
Just returning to this PR, I'm not sure that we want to demonstrate things like mustache sections? This feels like quite a lot of complexity, especially without comments to indicate what's going on. For our basic template questions it might be clearer to explicitly specify all the options? Although I agree that for the present case that's a lot of options, which would look horrible. Of course, for the current problem, we don't actually need any `server.py` anyway, because we could just pre-compute all the primes or factors up to 20 and then just hard-code the entire thing, which would be super clear. Maybe that's a better way to handle this?

Reply:
Yeah, I think it's clear that this is not the best question to demonstrate what we can do. Let's continue conversation on the other comment (https://github.com/PrairieLearn/PrairieLearn/pull/9878#discussion_r1619608379) so that we can first decide what we actually want this question to be? We can then decide if it makes sense to use dynamic lists like this or not.